### PR TITLE
salm

### PR DESCRIPTION
**Pull request by alisamimemmedli**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <p>test</p>
+    <p>salam</p>
```
Comment (alisamimemmedli) [inline index.html:11]: Write my name

Comment (alisamimemmedli) [reply index.html:11]: **Hello**

Comment (Riyad2021github) [reply index.html:11]: Ok

Comment (alisamimemmedli) [reply index.html:11]: Thankyou

Comment (alisamimemmedli) [reply index.html:11]: Are you bad boy

File: index.html [hunk 2]
```diff
@@ -9,5 +9,6 @@
 <body>
     <p>test</p>
     <p>salam</p>
+    <p>Alisami</p>
```
Comment (alisamimemmedli) [inline index.html:12]: write your name

**Pull request merged**